Updating docs per changes to staking procedures (#538)
subspace/subspace-docs: docs/farming-&-staking/staking/operators/register-operator.mdx

:::

> #### Register an operator using Subspace Staking interface (recommended)

Reviewer: ```suggestion
#### Register an operator using Autonomys Staking interface (recommended)
```

> #### Register an operator using Subspace Staking interface (recommended)

Reviewer 2: For some reason we have "Register an operator on domain" and "Register an operator using Subspace Staking interface (recommended)" one right after another. I think we can drop the latter.

> 1. Proceed to the [Subspace Staking portal](https://explorer.subspace.network/gemini-3h/staking) and connect your wallet.

Reviewer: ```suggestion
1. Proceed to the [Autonomys Staking portal](https://explorer.subspace.network/gemini-3h/staking) and connect your wallet.
```

Reviewer 2: Here too, would suggest referring to it as "Astral"

![NStaking-3](/img/doc-imgs/operators-staking/NStaking-3.png)

4. Select the account you'd like to use form the dropdown menu. You can click the button in the upper left corner to see available and locked (staked) token balances for the selected account. 

![NStaking-4](/img/doc-imgs/operators-staking/NStaking-4.png)

![NStaking-5](/img/doc-imgs/operators-staking/NStaking-5.png)

5. Proceed to the `Register Operator` tab.

![NStaking-6](/img/doc-imgs/operators-staking/NStaking-6.png)

6. Select the `domainId` you would like to be registered on.  For Stake Wars, Nova is Domain 0 and Auto-ID should be Domain 1 once it is available.

![NStaking-7](/img/doc-imgs/operators-staking/NStaking-7.png)

7. An important security change from Stake Wars one is that you are now required to have a Proof of signing key ownership signature.  This is derived from your seed phrase when creating the operator key.  
You can either select the file that contains your seed in the subfolder of **NODE_DATA_PATH** under `/domains/domainID/keystore` and then click the choose file and point to that file or you can choose the Proof with seed and manually enter the seed phrase.

> Once you click the Generate proof button it will automatically populate the `Signing key` and `Prof of signing key ownership signature` fields.  Enter the `Minimum Operator Stake`, `Amount to Stake`, `Nomination Tax` and then click `Next`.

Reviewer 2: ```suggestion
Once you click the Generate proof button it will automatically populate the `Signing key` and `Proof of signing key ownership signature` fields.  Enter the `Amount to Stake`, `Nominator Tax`, `Minimum Nominator Stake` and then click `Next`.
```

Reviewer 2: I think we should also add a note here that the minimum nominator stake [couldn't be lower than 1TSSC](https://academy.autonomys.net/subspace-protocol/decoupled-execution/staking#nomination-pools) if that change has made it into the code already.

![NStaking-8](/img/doc-imgs/operators-staking/NStaking-8.png)

:::info

> Make sure to either locate the signing key file generated on the previous **[Create operator key](#create-operator-key)** step or get use the key phrase that was displayed when the key was created and is also in that key file.

Reviewer: ```suggestion
Make sure to either locate the signing key file generated on the previous **[Create operator key](#create-operator-key)** step or use the key phrase that was displayed when the key was created and is also in that key file.
```

:::

> 7. Approve the request in the pop-up window. 

Reviewer 2: ```suggestion
8. Approve the request in the pop-up window. 
```

![NStaking-9](/img/doc-imgs/operators-staking/NStaking-9.png)

> 8. Congratulations, you're now registered as an **operator**!  You can click on the Manage My Operator tab to see your operator and get the Operator Id.

Reviewer 2: ```suggestion
9. Congratulations, you're now registered as an **operator**!  You can click on the Manage My Operator tab to see your operator and get the Operator Id.
```

![NStaking-10](/img/doc-imgs/operators-staking/NStaking-10.png)

:::info

> It can take up to 10 minutes for the operator to be registered and appear on the page.  It may take a day before your Stake amount is displayed under Total Stake in the grid.  Once you have your Operator Id you can restart your operator node and include the `--operator-id **OPERATOR_ID**` as a parameter. 

Reviewer: Is this 10 minutes for the squids to index? And I'm not sure where the day comes from here - it should just be an epoch transition I think.

Author: It has been about 18 hours since I registered my test operator and it still shows 0 tSSC.  That is why I assumed it would take a day.

> It can take up to 10 minutes for the operator to be registered and appear on the page.  It may take a day before your Stake amount is displayed under Total Stake in the grid.  Once you have your Operator Id you can restart your operator node and include the `--operator-id **OPERATOR_ID**` as a parameter. 

Reviewer 2: Not native speaker here, but "grid" sounds a bit weird. Table?
